database.json

```diff
@@ -1429,6 +1429,12 @@
     "repo": "is_url",
     "desc": "Check if a URL is absolute"
   },
+  "is_int": {
+    "type": "github",
+    "owner": "load1n9",
+    "repo": "denoint",
+    "desc": "checks if a number is an integer or not"
+  },
```

Comment:
put the repo in alphabetic order.

i.e. after the is_exe module & before the is_url module.